framework/src/modules/chain/bft/bft.js

```diff
@@ -136,23 +187,30 @@ class BFT extends EventEmitter {
 	/**
 	 * Load blocks into consensus manager fetching from storage
 	 *
-	 * @param {integer} fromHeight - The start height to fetch and load
-	 * @param {integer} tillHeight - The end height to fetch and load
+	 * @param {int} fromHeight - The start height to fetch and load
+	 * @param {int} tillHeight - The end height to fetch and load
 	 * @return {Promise<void>}
 	 */
-	async loadBlocks({ fromHeight, tillHeight }) {
+	async loadBlocksFromStorage({ fromHeight, tillHeight }) {
```

Comment:
Do we want to put this `blocks` related function inside of BFT? Can we move it to blocksLogic?

Reply:
Yes its BFT related logic. We can probably make it private, I kept it public just in case we came up with scenario to load blocks directly from storage somewhere.